lib/secure_headers/configuration.rb

```diff
@@ -104,6 +77,17 @@ def deep_copy(config)
         end
       end
 
+      # Private: Returns the internal default configuration. This should only
+      # ever be called by internal callers (or tests) that know the semantics
+      # of ensuring that the default config is never mutated and is dup(ed)
+      # before it is used in a request.
+      def default_config
```

Comment:
This private class method makes tests uglier, but it seems worthwhile to preserve hiding the actual default config from public callers.